app/assets/scss/application.scss

```diff
@@ -1,3 +1,6 @@
+// Reset styles on core HTML elements so that we have a clean base for DM styles to build on.
```

Comment:
Nice, might be nice to add something about browser added styles as those are the reason we have a reset stylesheet. Apart from that I like this

Reply:
Comment clarified. 👍